app/frontend/src/components/Button/Button.tsx

```diff
@@ -46,15 +49,24 @@ export default function Button<D extends ElementType = "button", P = {}>({
       setWaiting(false);
     }
   }
+  //height is fontSize * lineHeight, but they use rem
+  //so just use a &nbsp; to get the right height for loading
```

Comment:
Oh I see it from deleting the whitespace. Couldn't we enforce this with min height on the button itself? 🤔

Reply:
The problem is `theme.typography.button.fontSize` is in rem, so you can multiply it cleanly...

Reply:
Or you mean just set it manually?

Reply:
I set `minHeight` in the latest commit. But I kinda prefer the old way, this way can break if the button font size is made bigger.

Reply:
I think the height is actually `number of lines * line height`. If we explicitly set line height as well in the theme for buttons and get this `minHeight` referencing that, it'd fix the problem of having to update this in future?

Reply:
It can be either a multiplier or specified height. That would work as long as it's one line only (which it should be).

Reply:
Even with the non breaking white space, it'd go back down to one line when the button is loading, so I think it's fine. What do you mean by the multiplier option? I thought `minHeight` only works with length units like `px`/`rem` etc?

Reply:
If you don't put any units, it's a multiplier of fontSize (I actually didn't know you could specify the height exactly)

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/line-height

Reply:
Oh I see, yeah usually the multiplier is best as it scales properly with the user options, but I think it should be ok if we use `rem` to set the line height and reference that in `minHeight`. It should scale together with the font size since they are referencing the same root value unlike the `em` example shown in the article.

Reply:
Apparently Material UI got that set at 1.75, so I guess `1.75 * <font_size> rem` (works out to be `1.53125rem`) works?